packages/@divvi/mobile/src/public/types.tsx

```diff
@@ -178,6 +178,7 @@ export interface PublicAppConfig<tabScreenConfigs extends TabScreenConfig[] = Ta
 
   //
   networks?: {
+    enabledIds?: NetworkId[]
```

Comment:
nit: I know we're already in the `networks` group, but I feel this is more builder friendly.
```suggestion
    enabledNetworkIds?: NetworkId[]
```